README.md

```diff
@@ -10,15 +10,43 @@ The purpose of this repository is to allow easy management of all Pop!\_OS relat
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
+## Developer Resources
+For instructions on how to build the shell:
+	* [Cosmic, our current gnome-based shell](https://github.com/pop-os/cosmic). 
+	* [Cosmic-Epoch, the new Rust based shell](https://github.com/pop-os/cosmic-epoch).
+
 To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+Developer chat: https://chat.pop-os.org/
+
+## How to Make a Change
+First of all, thank you for taking interest in contributing to our community! Every bit helps, even if it's just contributing to our documentation. 
+
+### Find the Correct Repo
+Before you make a change, you need to find the relevant repository to make a contribution in. See the 'Developer Resources' section for help finding the correct one.  When in doubt, you can make an issue in this catch all repo. 
```

Comment:
There is an extra space before the last sentence.

Making an issue in this repo is more for when you have an _issue_ and you don't know where it goes, not necessarily for when you're trying to develop something and you don't know where it is, although I suppose someone might answer the question if it's asked here.